types/setting.go

```diff
@@ -59,4 +63,22 @@ var (
 		Required:    true,
 		ReadOnly:    true,
 	}
+
+	SettingDefinitionStorageOverProvisioningPercentage = SettingDefinition{
+		DisplayName: "Storage Over Provisioning Percentage",
+		Description: "The setting used for multiple disk scheduling, set the upper bound of disks",
+		Type:        SettingTypeInt,
+		Required:    true,
+		ReadOnly:    false,
+		Default:     "500",
+	}
+
+	SettingDefinitionStorageMinimalAvailablePercentage = SettingDefinition{
+		DisplayName: "Storage Minimal Available Percentage",
+		Description: "The setting used for multiple disk scheduling, set the bottom bound of disks",
```

Comment:
`If one disk's available capacity to it's maximum capacity in % is less than the minimal available percentage, the disk would become unschedulable until more space freed up.`